docs/user_guide/tutorials/02-collaboration/index.md

```diff
@@ -0,0 +1,221 @@
+
+## Motivation
+
+Collaboration is at the heart of effective GIS projects. Teams often include members with diverse backgrounds, skills, and areas of expertise. Without robust collaborative tools, it can become challenging to share insights, make real-time decisions, and maintain consistency across project workflows. JupyterGIS simplifies collaboration by providing real-time editing, annotations, and interactive features that allow teams to seamlessly integrate their work.
+
+:::{admonition} Objectives
+:class: seealso
+By following this tutorial, you will be able to:
+
+- Generate shareable links to invite collaborators.
+- Work together on GIS files and notebooks with live updates.
+- Use follow mode to monitor collaborator activities.
+- Add annotations and comments to provide context, ask questions, or share insights.
+  :::
+
+:::{admonition} Prerequisites
+:class: warning
+Before beginning this tutorial, JupyterGIS must be installed on your computer (see [Installation instructions](https://jupytergis.readthedocs.io/en/latest/user_guide/install.html)) or you can use an online version of JupyterGIS. Currently, real-time collaboration is not supported in [JupyterLite](https://jupytergis.readthedocs.io/en/latest/lite/lab/index.html?path=france_hiking.jGIS/).
+:::
+
+---
+
+## 1. Sharing Your Document
+
+### 1.1. Create Your GIS File
+
+- If you are using a local installation, run JupyterLab.
+
+```
+jupyter lab
+```
+
+- If you are using an online version, navigate to the JupyterGIS platform.
+- In the JupyterLab Launcher, scroll down to the **Other** section.
+- Click **GIS File** to open a blank canvas for your project.
+
+![New GIS File](images/new_gis_file.png)
+
+- Notice that you are given an anonymous username, which you can see in the upper right corner. Every user in the project will be assigned an anonymous username.
+
+![Username](images/username.png)
+
+### 1.2. Generate a Shareable Link
+
+- Click on the **Share** button in your interface on the upper right corner, then click on the **Copy Link** button.
+- Share this link with colleagues to invite them to your session.
+- If you are using a local installation, you can open a new browser and paste the link to simulate a different user.
+
+![Share](images/share.png)
+
+### 1.3. Confirm Collaborator Access
+
+- When your colleagues join using the link, their usernames appear in the top right corner.
+- This lets you know exactly who is working on the document.
+
+![Shared Users](images/shared_users.png)
+
+---
+
+## 2. Real-Time Collaboration on a GIS File
+
+### 2.1. Adding and Editing Layers
+
+- Open the layer panel and add a new layer to your GIS file.
+- The new layer appears immediately for all collaborators in your session.
+- Experiment by adjusting settings like opacity or color, each change is instantly visible to your collaborators.
```

Comment:
Perhaps from this bullet we could link to the intro tutorial which explains how to change symbology.